### DISCUSSION TITLE
how to add my path of file xlsx?

### DISCUSSION THREAD

**Discussion by SheikhElMoctarG** [exceljs/exceljs#2465] [Q&A]
I tried to get data from excel file using exceljs but I found an error : 
`Error: Can't find end of central directory : is this a zip file ? If it is, see https://stuk.github.io/jszip/documentation/howto/read_zip.html`

my code : 
`const ExelJS = require("exceljs");
const workhook = new ExelJS.Workbook();

workhook.xlsx.load("test.xlsx").then(()=>{
    const worksheet = workhook.getWorksheet(1);

}).catch((e)=> console.log("Error: "+ e.message));`

why is it happening and how to fix it  ??

**Comment by zurmokeeper** [comment]
@SheikhElMoctarG Can you provide your excel file?
